src/librbd/AioObjectRequest.cc

```diff
@@ -278,7 +288,7 @@ void AioObjectRead<I>::send() {
   librados::AioCompletion *rados_completion =
     util::create_rados_ack_callback(this);
   int r = image_ctx->data_ctx.aio_operate(this->m_oid, rados_completion, &op,
-                                          flags, nullptr);
+                                          flags, nullptr, this->m_trace ? this->m_trace.get_info() : nullptr);
```

Comment:
Nit: avoid going past 80 characters within a line. You can bring all the params for ```aio_operate``` down to the next line and indent by 2 spaces or use an intermediate variable for the trace info.